Makefile

```diff
@@ -347,6 +347,24 @@ data/static/.keep:
 	@mkdir -p data/static
 	@touch data/static/.keep
 
+migrations: ## WARNING: Temporary while prototyping
+	@echo "Start and wait for ${DB_HOST} to be up..."
+	@$(COMPOSE_UP_WAIT) -d ${DB_HOST}
+	@$(MANAGE) makemigrations -v 3 section
+.PHONY: migrations
+
+check: ## WARNING: Temporary while prototyping
+	@echo "Start and wait for ${DB_HOST} to be up..."
+	@$(COMPOSE_UP_WAIT) -d ${DB_HOST}
+	@$(MANAGE) check
+.PHONY: check
+
+check-migrations: ## WARNING: Temporary while prototyping
+	@echo "Start and wait for ${DB_HOST} to be up..."
+	@$(COMPOSE_UP_WAIT) -d ${DB_HOST}
+	@$(MANAGE) makemigrations --check --dry-run -v 3 section
+.PHONY: check
+
```

Comment:
IMO, we could keep `migrations` command at least, it can be useful.

Reply:
As seen with you, tasks `migrations` and `check-migrations` is part of this pull request, i've cleaned the `check` one